src/test/perf_helper.cc

```diff
@@ -0,0 +1,51 @@
+// -*- mode:C++; tab-width:8; c-basic-offset:2; indent-tabs-mode:t -*-
+// vim: ts=8 sw=2 smarttab
+/* Copyright (c) 2011 Facebook
```

Comment:
i might be wrong, but the copyright should also belong to stanford university, since the Perf tool comes from RAMCloud project.